optimade/server/routers/versions.py

```diff
@@ -0,0 +1,25 @@
+from fastapi import Request, APIRouter
+from fastapi.responses import Response
```

Comment:
Maybe we can subclass this and create our own CsvResponse that doesn't include the `application/json` media type?

Reply:
Maybe, I must admit I didn't try very hard to remove it. I was hoping the `PlainTextResponse` would work for this instead. I'll have another go.

Reply:
I actually don't know what the protocol would be here, the schema is for a JSON API, perhaps it should have a blank json response? Any thoughts @shyamd @fekad?

Reply:
You've done it! Yay!